pyblish_aftereffects/util.py

```diff
@@ -0,0 +1,48 @@
+import os
+import socket
+
+
+def send(cmd, port=None):
+
+    host = '127.0.0.1'
+    if not port:
+        port = int(os.environ["PYBLISH_AFTEREFFECTS_PORT"])
```

Comment:
This can throw an error if the environment variable is not declared. Better do `get()` and check again if there was none.

```python
port = port or os.getenv("PYBLISH_AFTEREFFECTS_PORT")
assert port, "No port number found"
```